.cursor/skills/run-all-checks/SKILL.md

```
@@ -0,0 +1,165 @@
+- `-c, --code`: Only ruff, mypy, pytest.
+- `-d, --docs`: Only Sphinx build and PyMarkdown scan.
+- `-m, --markdown`: Only PyMarkdown scan.
+- `-s, --sequential`: Run code and docs sequentially (easier to read output).
+- `-p, --parallel`: Run code and docs in parallel (the default).
+- `-h, --help`: Show usage.
+
+Set `VENV` or `VENV_PATH` to point to the virtual environment if it is not at `./venv`.
+
+## Execution Workflow
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)